ErabliereIU/src/rappel/rappels.component.html

```diff
@@ -0,0 +1,6 @@
+<div class="row">
+    <h2>Rappels</h2>
+    <div class="col-12 mb-2" *ngFor="let note of todayReminders; let i = index">
+      <app-rappel [note]="note" [index]="i"></app-rappel>
```

Comment:
Pareil que pour le ngIf, utilise plutôt [le décorateur for](https://angular.io/api/core/for), c'est une solution plus moderne et plus performante.